### PR TITLE
Navigation component

### PR DESCRIPTION
**Pull request by sahiba0915**
hello team please review my code.

Live preview: https://navigation-component-cl.netlify.app/component/navigation/navigation.html

Files to check:
1. navigation.html
2. navigation.css
3. src.css

### CODE REVIEW

**Review by chanchal-panpaliya — COMMENTED** [review]
flex
use flex for alignment .  the class name is consistent and code is readable .

File: component/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  --secondary-text: #747171;
+  --primary-color: #f9a826;
+  --secondary-color: #ffffff;
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Poppins", sans-serif;
+  --primary-alert: #80b918;
+  --success-alert: #dee2e6;
+  --info-alert: #4cc9f0;
+  --warning-alert: #d62828;
```
Comment (ivinayakg) [inline component/navigation/navigation.css:18]: just my personal preference, use as less CSS variables as possible especially colors, you won't remember them.

**Review by ivinayakg — APPROVED** [review]
Your naming is spot on, very precise zero spaces for errors, great work 👍
P.S - don't add files that are not related to the PR makes the PR less complicated 🙂

File: component/navigation/navigation.css (hunk 1 above)
Comment (sahiba0915) [reply component/navigation/navigation.css:18]: thanks will surely work on the mentioned points

File: component/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  --default-color: #000000;
+  --pink: #e76f51;
+  --border: rgba(0, 0, 0, 0.3);
```
Comment (akash-1318) [inline component/navigation/navigation.css:21]: Use hex color code here also .

**Pull request merged**